source/index.html.md

```diff
@@ -322,6 +322,12 @@ error categories, the last two digits define specific errors.
 * 1202: We cannot send an SMS to the number provided (out of jurisdiction)
 * 1299: Unexpected error when trying to send SMS
 
+### OTP related errors: 13 (425 error)
+* 1301: OTP hash value isn't valid!
+* 1302: Duplicated token save attempt, there's another valid OTP!
```

Comment:
**Question:** what scope is this?
Is it ever going to be sent to anybody interacting with the Bitso API for pulling balances, generating orders or withdrawing?
Is it something that will be used for specific Bitso projects such as the app?

Reply:
@thenewversion I created it for a specific case in conversions, but seeing that we don't have 404 errors or related I made it more generic thinking that it can be used for general purposes

Reply:
1. no, is not going to be sent when using the Bitso API (the public one, only for the private/protected one)
2. yep

Reply:
In that case, I would suggest labelling them `reserved for internal use` or similar.
Looking at it, they should not even be mentioned as there is nothing else in that group

Reply:
🤔 ... mmm well yes, can take the whole section out ... or just label it